packages/react-router/src/hooks/redirect.ts

```diff
@@ -1,12 +1,16 @@
 import {useEffect} from 'react';
+import {NavigateOptions} from 'router';
 import type {NavigateTo} from '../types';
 
 import {useNavigate} from './navigate';
 
-export function useRedirect(to: NavigateTo) {
+export function useRedirect(
+  to: NavigateTo,
+  {replace = true, relativeTo}: NavigateOptions = {},
```

Comment:
We shouldn't include the `replace` option, because only `replace: true` makes sense for a redirect (otherwise, this is what I was looking for, yeah :)

Reply:
oh yeah 🤦